### ISSUE TITLE
[REGRESSION] `conan info` does not work correctly with 1.17.0

### ISSUE THREAD

**Issue by DoDoENT** [conan-io/conan#5479]
After upgrading Conan from `1.16.1` to `1.17.0`, `conan info` command stopped working.

Here is the output for one package with Conan v1.16.1:

```
$ conan info AndroidNdk/r20@microblink/stable
AndroidNdk/r20@microblink/stable
    ID: 743cf0321be3152777da4d05247a66d1552e70a2
    BuildID: None
    Remote: microblink=http://artifactory.microblink.com/artifactory/api/conan/conan-local
    URL: https://developer.android.com/ndk/downloads/
    Recipe: Cache
    Binary: Cache
    Binary remote: microblink
    Creation date: 2019-06-18 11:00:55
```

The same command with Conan `v1.17.0` gives the following:

```
conan info AndroidNdk/r20@microblink/stable
ERROR: AndroidNdk/r20@microblink/stable: 'options.enable_testing' doesn't exist
Possible options are []
```

I haven't given any option to `conan info`, so where does it get it from? Furthermore, if same command is given from empty folder, then it works. My guess is that it is somehow related to executing this command from folder that contains `conan.lock` for completely different package (which does have `enable_testing` option).

Now, why does `conan info` read lockfile even if `--lockfile` option is not given.

I've tried removing the `conan.lock` file, and then something even weirder happens:

```
$ rm conan.lock
[dodo@knight-rider: Debug]$ conan info AndroidNdk/r20@microblink/stable
ERROR: Missing lockfile in: /my/current/folder/containing/cmake/cache/and/conan/installation/for/different/package
```

However, deleting file `graph_info.json` from current folder makes `conan info` work, however I do not want to do that - I am not interested for information about conan package installed in current folder (if I were, I would call `conan info /path/to/conanfile.py`).

**Comment by lasote** [comment]
Reproduced

**Comment by memsharded** [comment]
Hi @DoDoENT 

Thanks for this report! I am proposing this PR to fix it: https://github.com/conan-io/conan/pull/5481, to be released in 1.17.1 if possible.

**Comment by lasote** [comment]
Merged, to be released at 1.17.1
